core/drivers/stm32_etzpc.c

```diff
@@ -0,0 +1,416 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2015-2017, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2017-2018, STMicroelectronics
+ */
+
+/*
+ * STM32 ETPZC as as a firewall on SoC peripheral interfaces and internal
+ * memories. Driver expects a single instance of the controoler in a SoC.
+ *
+ * When CFG_DT is enable, the ETZPC configuration is probed from a secure DTB,
+ * currently the embedded DTB only.
+ * When CFG_DT is disable, the platform shall initialize the ETZPC device,
+ * providing the interface register base address, through stm32_etzpc_init().
+ *
+ * Driver exports an API to access its resources. Platforms are in change of
+ * ensuring the consitency of the ETZPC active configuration and the software
+ * assumptions.
+ *
+ * Driver registers a PM callback. When resuming, the configured permissions
+ * are restored for securable peripherals and securable memories.
+ */
+
+#include <assert.h>
+#include <drivers/stm32_etzpc.h>
+#include <dt-bindings/soc/st,stm32-etzpc.h>
+#include <kernel/dt.h>
+#include <kernel/generic_boot.h>
+#include <initcall.h>
+#include <io.h>
+#include <keep.h>
+#include <kernel/panic.h>
+#include <kernel/pm.h>
+#include <mm/core_memprot.h>
+#include <stdint.h>
+#include <util.h>
+
+#include <drivers/serial.h>
+
+#ifdef CFG_DT
```

Comment:
The `#ifdef` can safely be removed I think (and the line moved up with the others)

Reply:
Well `CFG_DT` is disabled, **core/lib/libfdt/include** is not added to the include path hence **libfdt.h** not visible.

Reply:
OK, let's keep the `#ifdef` then.